ipaullly/sendIT
# [#161980170] Removes 505 error for invalid user/parcel id

#### What does this PR do?
catches the instances where a user may submit a wrong id when canceling or retrieving orders by a specific user_id 
#### Description of Task to be completed?
- Define a `method` in parcels model to check whether `id` passed into it exists in the database
- add logic to view functions to raise an error message when invalid id is passed through the URL 
#### How should this be manually tested?
access the API documentation at https://documenter.getpostman.com/view/4014888/RzZCCwun, press the `Run in Postman` at the top right corner of the window. Try to send the endpoints that require a user/parcel id with an invalid id and an error message appears promptly
